### DISCUSSION TITLE
Finding the source of a string in the `renovatebot/renovate` code

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#12537] [Request Help] [answered]
## What do I want to do?

The last checkbox in the Dependency Dashboard says:

> - [ ] Check this box to trigger a request for Renovate to run again on this repository

I want to replace the verb `Check` with `Click`, like this:

> - [ ] Click this box to trigger a request for Renovate to run again on this repository

This way all our strings use the verb `Click` instead of `Check`, which is clearer semantically.

## What I've tried myself:

I've tried these search terms in VSCode on the code in `renovatebot/renovate`:

- `Check this box`
- `trigger a request`
- `Renovate to run again`

None of those resulted  in a hit for me. Is the source for this string somewhere else maybe?

## Can you help?

Do you have any ideas? I'm probably doing my search wrong, I'd expect the code for this string to live in this repository. 🙈

**Comment by rarkins** [comment, marked as answer]
This is probably a string which is added *by config* from the app itself. It's not default in Renovate because by default clicking on a box in dependency dashboard isn't going to trigger the CLI to do anything at all.. it won't see it until the next time it's running.. in which case the checkbox was pointless (and probably aggravating because it did nothing)

**Reply by HonkingGoose** [reply]
> This is probably a string which is added _by config_ from the app itself.

Ah, that's why I cannot find it in the `renovatebot/renovate` open source code. 😄 

>  It's not default in Renovate because by default clicking on a box in dependency dashboard isn't going to trigger the CLI to do anything at all.. it won't see it until the next time it's running.. in which case the checkbox was pointless (and probably aggravating because it did nothing)

Of course, if you run the app from the CLI, you don't _need_ any checkbox to run again, as you can just run the CLI tool again. 😄 

---

@rarkins Do you maybe want to change the verb from `Check` to `Click` in the GitHub hosted app code? Or do you want to leave it be?
